modules/redis-cluster/metadata.yaml

```diff
@@ -146,6 +186,7 @@ spec:
       - networkconnectivity.googleapis.com
       - compute.googleapis.com
       - memorystore.googleapis.com
+      - cloudkms.googleapis.com
     providerVersions:
       - source: hashicorp/google
-        version: ">= 6.0, < 7"
+        version: ">= 4.74.0, < 7"
```

Comment:
This seems odd? It should be > 6.22 right?

Reply:
Apparently metadata updated by cft command grabbed version from root module for all sub-modules. I updated all of them manually.